### DISCUSSION TITLE
Pop-Up Notifications

### DISCUSSION THREAD

**Discussion by Spuner** [Rem0o/FanControl.Releases#2381] [Ideas]
Please add to the settings the ability to disable the pop-up notification about changing profiles. If different profiles have been changed many times, then this notification is shown there many times. You can't imagine how annoying this is.

![2024-03-18 02_08_32-Fan Control V184 (Profile 1 - Auto Fans Control json)](https://github.com/Rem0o/FanControl.Releases/assets/30773815/04f90518-b725-4ae6-a79d-ed010219411c)

**Comment by Rem0o** [comment]
Is it that bad? It doesn't prevent you from interacting with the software in any way no?

**Reply by Spuner** [reply]
He's very annoying. Honestly, I don't understand the point of this notice. If profiles change many times, then this notification only shows which profiles were used and that’s it.
